src/MediaCT/phpmd.xml

```diff
@@ -1,4 +1,10 @@
 <?xml version="1.0"?>
+<!--
+/**
+ * Copyright Youwe. All rights reserved.
+ * https://www.youweagency.com
+ */
+-->
 <ruleset name="MediaCT PHPMD rules"
```

Comment:
Why not also update these? (replacing MediaCT with Youwe). Is it because you made this one deprecated?

Reply:
yes, i don't want people to have a broken testing suite once this has been added.